generators/python-v2/ast/src/PythonFile.ts

```diff
@@ -1,26 +1,41 @@
 import { AstNode } from "./core/AstNode";
+import { Comment } from "./Comment";
 import { Writer } from "./core/Writer";
 import { Reference } from "./Reference";
 import { ModulePath } from "./core/types";
+import { StarImport } from "./StarImport";
 
 export declare namespace PythonFile {
     interface Args {
         /* The path of the Python file relative to the module */
         path: ModulePath;
+        /* The list of statements in the Python file. More can be added following initialization. */
+        statements?: AstNode[];
         /* Whether or not this represents the root of a Python module */
         isInitFile?: boolean;
+        /* Any comments that should be at the top of the file */
+        topOfFileComments?: Comment[];
+        /* Any star imports that should be included */
+        starImports?: StarImport[];
```

Comment:
nit: Thoughts on just naming the field `imports` even if it's typed as `StarImport`?

Reply:
As a consumer of this package, this alternative naming might get me spooked and make me think that I’m supposed to explicitly include all imports, whereas most importing is handled for me via References.

I’m down for an alternative name though. Previously you had suggested “sideEffectImports” - wdyt about that?

Reply:
I'll roll with `imports` and if we think of something better later, we can change it.